packages/babel-plugin/src/components-and-actions.ts

```diff
@@ -1,12 +1,15 @@
 import * as Babel from "@babel/core";
+import * as nodePath from "path";
```

Comment:
I think this should be called `filePath` or just `path` (and rename the babel `path` to `nodePath`). When I think of node in a transpiler I think of an AST node, node NodeJS

Reply:
Great point! How did I miss this!

Renamed to `pathLib`. Is that any better?